wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -267,6 +268,77 @@ class BeverageChoiceBlock(blocks.ChoiceBlock):
         )
 
 
+class TestRawHTMLBlock(unittest.TestCase):
+    def test_get_default_with_fallback_value(self):
+        default_value = blocks.RawHTMLBlock().get_default()
+        self.assertEqual(default_value, '')
+        self.assertTrue(isinstance(default_value, SafeText))
+
+    def test_get_default_with_none(self):
+        default_value = blocks.RawHTMLBlock(default=None).get_default()
+        self.assertEqual(default_value, '')
+        self.assertTrue(isinstance(default_value, SafeText))
+
+    def test_get_default_with_empty_string(self):
+        default_value = blocks.RawHTMLBlock(default='').get_default()
+        self.assertEqual(default_value, '')
+        self.assertTrue(isinstance(default_value, SafeText))
+
+    def test_get_default_with_nonempty_string(self):
+        default_value = blocks.RawHTMLBlock(default='<blink>BOOM</blink>').get_default()
+        self.assertEqual(default_value, '<blink>BOOM</blink>')
+        self.assertTrue(isinstance(default_value, SafeText))
+
+    def test_serialize(self):
+        block = blocks.RawHTMLBlock()
+        result = block.get_prep_value(mark_safe('<blink>BOOM</blink>'))
+        self.assertEqual(result, '<blink>BOOM</blink>')
+
+    def test_deserialize(self):
+        block = blocks.RawHTMLBlock()
+        result = block.to_python('<blink>BOOM</blink>')
+        self.assertEqual(result, '<blink>BOOM</blink>')
+        self.assertTrue(isinstance(result, SafeText))
```

Comment:
Ah, yes - it did feel ugly doing assertTrue all the time there. (And anyhow, it needs to be SafeData for Python2 compatibility...)